README.md

```diff
@@ -1,42 +1,89 @@
+[![InstantSearch.js logo][logo]][website]
+
+InstantSearch.js is a library for building blazing fast search-as-you-type search UIs with [Algolia](algolia-website).
+
+InstantSearch.js is part of the InstantSearch family: 
+**InstantSearch.js** 
+| [React InstantSearch (web)][react-instantsearch-github]
+| [Vue InstantSearch (web)][vue-instantsearch-github]
+| [InstantSearch Android (native)][instantsearch-android-github]
+| [InstantSearch iOS (native)][instantsearch-ios-github].
```

Comment:
Should we add as well the same link for `react-native` support as well? It adds keywords on the README, would not hurt I suppose 👍

Reply:
I will also update the other projects afterwards then, as per @vvo proposal